tests/datavolume_test.go

```diff
@@ -0,0 +1,63 @@
+package tests_test
+
+import (
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
+
+	"kubevirt.io/kubevirt-ansible/tests"
+	kubev1 "kubevirt.io/kubevirt/pkg/api/v1"
+	ktests "kubevirt.io/kubevirt/tests"
+)
+
+// template parameters
+const rawDataVolumeVMFilePath = "tests/manifests/template/datavolume-vm.yml"
+const rawDataVolumeVMIFilePath = "tests/manifests/template/datavolume-vmi.yml"
+const rawDataVolumeFilePath = "tests/manifests/template/datavolume.yml"
+
+var _ = Describe("DataVolume Integration Test", func() {
+	var dataVolumeName, vmName, dstDataVolumeFilePath, url, dstVMIFilePath string
+
+	Context("Datavolume with VM", func() {
+		BeforeEach(func() {
+			dataVolumeName = "datavolume1"
+			vmName = "test-vm-i"
+			dstDataVolumeFilePath = "/tmp/test-datavolume-vm.json"
+			url = "https://download.cirros-cloud.net/0.4.0/cirros-0.4.0-x86_64-disk.img"
+		})
+
+		It("Creating VM and start VMI will be success", func() {
```

Comment:
I think we should also verify that restarting the VM is successful. 

Here's an example.
https://github.com/kubevirt/kubevirt/blob/master/tests/datavolume_test.go

We test starting and stopping the VM multiple times.

Reply:
okay, no problems will add it.